dhall/benchmark/normalize/Main.hs

```diff
@@ -0,0 +1,21 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Main where
+
+import Gauge (defaultMain, bgroup, bench, nf)
+
+import qualified Gauge
+import qualified Data.Text as T
+import qualified Data.Text.IO as TIO
+import qualified Dhall.Core as Dhall
+import qualified Dhall.Parser as Dhall
+
+benchExprFromText :: String -> T.Text -> Gauge.Benchmark
+benchExprFromText name expr =
+    bench name $ nf (Dhall.exprFromText "(input)") expr
```

Comment:
So far this benchmarks only parsing. You can find the remaining steps e.g. here:

https://github.com/dhall-lang/dhall-haskell/blob/6ccf08d6fb28ee35cf9e938cdce225e65ea84df6/dhall/src/Dhall/Main.hs#L648-L654

Reply:
Yes! Would you like those as well? From profiling, I found that parsing is sort-of benchmark.